settings.gradle.kts

```diff
@@ -49,6 +49,11 @@ include(
   "clients:client-python",
   "clients:cli"
 )
+if (gradle.startParameter.projectProperties["enableFuse"]?.toBoolean() ?: false) {
+  include("clients:filesystem-fuse")
+} else {
+  println("Skipping Gvfs-fuse tasks since -PenableFuse=true is not enabled.")
```

Comment:
Better change to: "Skipping filesystem-fuse module since enableFuse is set to false."